nixos/modules/services/web-apps/komga.nix

```diff
@@ -66,9 +104,17 @@ in
         };
       };
 
+      systemd.tmpfiles.settings."10-komga" = {
+        ${cfg.stateDir}.d = {
+          inherit (cfg) user group;
+        };
+        "${cfg.stateDir}/application.yml"."L+" = {
```

Comment:
TIL `L+` [symlink to [re]create](https://www.freedesktop.org/software/systemd/man/latest/tmpfiles.d.html)